boosters/node-beginner/tests/config/setup.js

```diff
@@ -0,0 +1,3 @@
+const db = require("../../models");
+
+module.exports = database => data => async () => db[database].bulkCreate(data);
```

Comment:
For new-comers to the booster, deeply nested arrow functions might be difficult to interpret. We may want to re-write this for ease of understanding.

Also, does the `bulkCreate` call need an `await` keyword in front to ensure that the operation completes?

Reply:
Makes sense. I'll change it to only be one level deep